ethcore/src/machine.rs

```diff
@@ -377,8 +377,8 @@ impl EthereumMachine {
 	}
 
 	/// If this machine supports wasm.
-	pub fn supports_wasm(&self) -> bool {
-		self.params().wasm
+	pub fn supports_wasm(&self, blocknumber: BlockNumber) -> bool {
```

Comment:
Do we need a separate public method for that? perhaps use `Schedule` directly on the call site.